BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -187,9 +187,16 @@ - (void)configurationLoaded:(__unused BTConfiguration *)configuration error:(NSE
                 }
             }
 
-            // Always add Cards
-            [activePaymentOptions addObject:@(BTUIKPaymentOptionTypeUnknown)];
-            
+            NSArray *supportedCardTypes = [self.configuration.json[@"creditCards"][@"supportedCardTypes"] asArray];
+            for (NSString *supportedCardType in supportedCardTypes) {
+                BTUIKPaymentOptionType paymentOptionType = [BTUIKViewUtil paymentOptionTypeForPaymentInfoType:supportedCardType];
+                if ([BTUIKViewUtil isPaymentOptionTypeACreditCard:paymentOptionType]) {
```

Comment:
Ah, got it. I want to say those card types are also returned in the specific configuration blocks for A*Pay payment methods, so that might be something we can optimize in the future.

Looks good!